### PR TITLE
Add merged objects section to getting started docs

### LINKED ISSUE #216: Add "Working with merged objects" to getting started section
We would like to demonstrate how you can use merged objects for integration in the _Getting started with an ScPCA dataset_ section. My IA suggestion: place this directly after _Working with processed objects_ in `docs/getting_started.md`.

### PR DESCRIPTION
**Pull request by sjspielman**
Closes #216

Here, I've added some light "getting started" docs for merged objects. I provide OSCA links for DE and integration, as well as an integration benchmark paper, for more resources. I also specifically show reading objects in and subsetting to libraries of interest.

For review:
- Is there additional content we should consider?
- Should any of this be reorganized or differently emphasized?

### CODE REVIEW

File: docs/getting_started.md [hunk 1]
```diff
@@ -328,6 +328,61 @@ See these resources for more information on automated cell type annotation:
 - [Assigning cell types with `SingleR`](https://bioconductor.org/books/release/SingleRBook/)
 - [Cell type assignment chapter in Orchestrating Single Cell Analysis](https://bioconductor.org/books/3.17/OSCA.advanced/cell-cycle-assignment.html)
 
+
+## Working with a merged ScPCA object
+
+Merged ScPCA objects contain all information present in `processed` objects for a given ScPCA project.
```
Comment (allyhawkins) [inline docs/getting_started.md:334]:
```suggestion
Merged ScPCA objects contain all information found in `processed` objects for all individual libraries that compile a given ScPCA project.
```

File: docs/getting_started.md [hunk 2]
```diff
@@ -328,6 +328,61 @@ See these resources for more information on automated cell type annotation:
...
+For more information on how these objects were prepared, see {ref}`the section on merged object preparation<processing_information:merged objects`.
+
+Merged object can facilitate joint analysis on multiple samples at the same time, for example to perform [differential expression analysis](https://bioconductor.org/books/3.16/OSCA.multisample/multi-sample-comparisons.html) or integration (i.e., batch correction).
+**However, please be aware that data in merged objects has not been integrated/batch-corrected.**
+If you are interested in performing integration, we recommend consulting with the following resources:
```
Comment (allyhawkins) [inline docs/getting_started.md:339]: I think I would expand on this and seprately mention things you can do with merged objects vs. integration. So I would first talk about why you would want to use a merged object - differential expression analysis or any analysis where you are comparing gene level metrics between samples. I would even provide some links to example analyses. 

Then, I would mention that there are other types of analysis that require integration and that we do not perform integration. I would specifically mention that anything where you are comparing cell level metrics (clusters) may need to account for batch correction.

**Review by allyhawkins — COMMENTED** [review]
Thanks for getting this started. I think it's important that we emphasize the difference between things you can do with these merged objects and things that may require integration or batch correction. Right now that's all in one sentence and I think it's important we expand it and be clear about what you would use these objects for.

File: docs/getting_started.md (hunk 2 above)
Comment (sjspielman) [reply docs/getting_started.md:339]: >  I would even provide some links to example analyses.

Are you thinking, for example, actual papers which have performed the analyses? Or more general resources about those sorts of analyses?

Comment (sjspielman) [reply docs/getting_started.md]: Took a first stab here, let me know what you think! Heading in the right direction?
https://github.com/AlexsLemonade/scpca-docs/pull/230/commits/1d45d8026a8e0630d07500458eae10dbcd85b1ba

File: docs/getting_started.md [hunk 3]
```diff
@@ -328,6 +328,81 @@ See these resources for more information on automated cell type annotation:
...
+  - [Differential expression analysis chapter in Orchestrating Single Cell Analysis](https://bioconductor.org/books/3.17/OSCA.multisample/multi-sample-comparisons.html)
+  - [Differential gene expression analysis chapter in Single-cell Best Practices](https://www.sc-best-practices.org/conditions/differential_gene_expression.html)
+  - [10x resource: Differential gene expression analysis in scRNA-seq data between conditions with biological replicates](https://www.10xgenomics.com/resources/analysis-guides/differential-gene-expression-analysis-in-scrna-seq-data-between-conditions-with-biological-replicates)
+- Gene-set enrichment analysis
+  - [Gene set enrichment and pathway analysis chapter in Single-cell Best Practices](https://www.sc-best-practices.org/conditions/gsea_pathway.html)
+
+
+By contrast, analyses which compare _cell-level metrics_ among different samples require an **integrated/batch-corrected** object.
+These types of analyses include, for example, [differential cell abundance](https://bioconductor.org/books/3.17/OSCA.multisample/differential-abundance.html) or [differential cell composition analysis](https://www.sc-best-practices.org/conditions/compositional.html).
```
Comment (allyhawkins) [inline docs/getting_started.md:370]: I'm torn on if we want to mention clustering. Specifically, that clusters should be re-computed after merging. But we also don't provide clusters on the merged object, so maybe it's not worth mentioning. You decide!

File: docs/getting_started.md [hunk 4]
```diff
@@ -328,6 +328,81 @@ See these resources for more information on automated cell type annotation:
...
+
+- [Correcting batch effects chapter in Orchestrating Single Cell Analysis](https://bioconductor.org/books/3.17/OSCA.multisample/integrating-datasets.html)
+- [Correction diagnostics chapter in Orchestrating Single Cell Analysis](https://bioconductor.org/books/3.17/OSCA.multisample/correction-diagnostics.html)
+- [Integration chapter in Single-cell Best Practices](https://www.sc-best-practices.org/cellular_structure/integration.html)
+- [Luecken _et al._ (2021) Benchmarking atlas-level data integration in single-cell genomics](https://doi.org/10.1038/s41592-021-01336-8)
+
+
+### Subsetting libraries in the merged object
+
+You may wish to only work with a subset of libraries present in the merged object.
```
Comment (allyhawkins) [inline docs/getting_started.md:383]: I think we should mention that there is metadata in the object that could be useful to subsetting. For example, if you want to look at all samples of a specific diagnosis, tissue type, etc then you could use those columns to subset.

**Review by allyhawkins — COMMENTED** [review]
Definitely in the right direction! I like this layout a lot and just had a few smaller comments.

File: docs/getting_started.md (hunk 3 above)
Comment (sjspielman) [reply docs/getting_started.md:370]: > You decide!

Oh no this is a hard one to decide on 😂 , maybe I'll bring it up in pre-planning...

File: docs/getting_started.md (hunk 4 above)
Comment (sjspielman) [reply docs/getting_started.md:383]: Great idea!

**Comment by sjspielman** [comment]
I added some content about using metadata to subset, which necessarily includes one TODO to link in a section that hasn't been written yet. I also included a dummy example - what do you think of it?

Also, I ignored `.DS_Store`. I was surprised nobody had been sufficiently annoyed by that yet!

File: docs/getting_started.md [hunk 5]
```diff
@@ -328,6 +328,106 @@ See these resources for more information on automated cell type annotation:
...
+
+- Differential expression analysis
+  - [Differential expression analysis chapter in Orchestrating Single Cell Analysis](https://bioconductor.org/books/3.17/OSCA.multisample/multi-sample-comparisons.html)
+  - [Differential gene expression analysis chapter in Single-cell Best Practices](https://www.sc-best-practices.org/conditions/differential_gene_expression.html)
+  - [10x resource: Differential gene expression analysis in scRNA-seq data between conditions with biological replicates](https://www.10xgenomics.com/resources/analysis-guides/differential-gene-expression-analysis-in-scrna-seq-data-between-conditions-with-biological-replicates)
+- Gene-set enrichment analysis
+  - [Gene set enrichment and pathway analysis chapter in Single-cell Best Practices](https://www.sc-best-practices.org/conditions/gsea_pathway.html)
+
+
+By contrast, analyses which compare _cell-level metrics_ among different samples require an **integrated/batch-corrected** object.
```
Comment (allyhawkins) [inline docs/getting_started.md]: Hmmm "require" might be a strong word. I think we should say that if you want to decrease batch effects then it's recommended to integrate? 
```suggestion
By contrast, it is recommended to control for batch effects when performing analyses that compare _cell-level metrics_ among different samples. 
In order to do this, a **integrated/batch-corrected** object should be created.
```

**Review by allyhawkins — COMMENTED** [review]
I think the scope of the metadata additions is good, but I'm going to wait to approve until we decide what should be mentioned regarding clustering.

**Comment by sjspielman** [comment]
Added the "you may wish to cluster" bit as discussed! Let me know any more feedback on wording.

**Review by allyhawkins — APPROVED** [review]
Looks good!

**Pull request merged**